### ISSUE TITLE
Error: Command failed: @example.local does not specify a remote-host

### ISSUE THREAD

**Issue by kostajh** [elliotttf/drush-reloadp#1]
I'm trying to sync a database from a staging site to my local environment, but get this error:

> Error: Command failed: @example.local does not specify a remote-host

My local alias looks like:

``` php
$aliases['local'] = array(
  'root' => $base_path . '/docroot',
  'uri' => 'http://local.example.org',
  'db-url' => 'mysql://root@localhost/example_local',
);
```

I've tried with both Drush 6 and Drush 7. Any ideas?

**Comment by elliotttf** [comment]
The script actually assumes it needs to SSH into both the source and destination. Smarter parsing of the alias could fix that but as a workaround you can define a remote host (as localhost) and just make sure you can ssh to your local machine:

``` php
$aliases['local'] = array(
  'root' => $base_path . '/docroot',
  'uri' => 'http://local.example.org',
  'db-url' => 'mysql://root@localhost/example_local',
  'remote-host' => 'localhost',
  'remote-user' => 'you',
);
```

**Comment by kostajh** [comment]
Hmm, yeah I don't want to enable SSH on my local machine.

Perhaps in getCores you could check if the docroot specified in the destination alias exists, and only if not, attempt the SSH.

**Comment by elliotttf** [comment]
Yup, this is a bug that needs to be addressed, I was just pointing out a workaround that will solve your issue until it can be addressed the correct way.

Feel free to open a PR against this if you have time, I won't be able to look at this until next week at the earliest.

**Comment by elliotttf** [comment]
@kostajh I believe this has been fixed in v0.1.3 but I don't have a local site to test with right now. Want to be a guinea pig?

cc @mateu-aguilo-bosch

**Comment by kostajh** [comment]
@elliotttf yes that appears to be fixed. Thanks for doing that!
